content/articles/premium-domains.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Premium Domains
+excerpt: This page explains the concept of premium domains and provides help on how to register one with us.
+categories:
+- Domains
+---
+
+# Premium Domains
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## What is a Premium Domain?
+
+A premium domain is a domain that has been considered of special value by a registrar. Premium domains generally are available in the new gTLDs domain space. They are usually more expensive that regular domains, prices going up to several thousand dollars for one domain in some cases.
+
+## What makes a Premium Domain valuable
+
+Premium domains appeal to specific businesses or groups of people providing them very with useful, specific domain names for their activities. Other times they are just popular keywords or a very well known combination of name and domain. This is better illustrated with an example: 
+
+For a company such us Heineken, owning the domain heineken.beer is a very powerful and concise way to market its activity, which as you may know is making and selling beers.
```

Comment:
Rather than going with a brand, I'd use a different example. For example, you can mention that combination of single-names and TLDs that have very close relations together (e.g. `ruby.codes` or `madrid.love`) are generally premium domains as they are considered having globally recognized value due to the wording itself.

Another mention I think is worth is that 2 and 3 letters new gTLD domains are generally considered premium domains regardless.